bcipy/parameters/devices.json

```diff
@@ -120,5 +120,17 @@
         "excluded_from_analysis": ["device_ts", "system_ts", "left_pupil", "right_pupil"],
         "status": "active",
         "static_offset": 0.0
+    },
+    {
+        "name": "Switch",
+        "content_type": "MARKERS",
```

Comment:
I had missed this before, but should this be `MARKER`? We would register one switch or marker stream at a time, similar to the other devices. 

If this is not a continuous stream, like would be the case for LSL markers, I'm not sure it makes sense to be registered here. If it were, we would like to know the origin (ex. psychopy_switch, lsl_marker_stream) to help understand how to register any changes configured here.